docs/guides/mozilla-sops.md

```diff
@@ -131,10 +131,24 @@ AWS IAM Role example:
 }
 ```
 
+Azure:
+
 When using Azure Key Vault you need to authenticate the kustomize controller either by passing
 [Service Principal credentials as environment variables](https://github.com/mozilla/sops#encrypting-using-azure-key-vault)
 or with [add-pod-identity](https://github.com/Azure/aad-pod-identity).
 
+GCP Role:
+
+When using GCP, create a service account with the role `Cloud KMS CryptoKey Encrypter/Decrypter`, create an bind an IAM Rolebinding 
```

Comment:
Split up these with bullet points please. Also "create an bind an IAM Rolebinding between the GCP service account to the `default` service account of the `flux-system`" doesn't sound ok to me, GCP docs have no such language, they use IAM policy binding in the context of workload identity. Does this require workloads identity?

Reply:
Yes, it does. I will change the wording